host/3.0/node-build.yml

```diff
@@ -29,6 +31,14 @@ steps:
     continueOnError: false
     env:
       pswd: $(dockerPassword)
+  
+  - bash: |
+      set -e
+      HOST_VERSION=3.0.15584
```

Comment:
Tagging @TsuyoshiUshio - for more details on what needs to be updated in the build pipelines

- `HOST_VERSION` should be an input parameter
- Need to add a new pipeline to build function base image and push it mcr

Reply:
One downside of having `HOST_VERSION` as an input parameter to the build task, rather than a part of a commit, is that it's more difficult to run CI automatically on it. You will have to either extract it from the PR branch name, or run it manually setting the value. This PR seems to still have it in all the docker files, as well as here. Not sure what the end goal is.